README.md

```diff
@@ -25,13 +25,33 @@ docker exec --interactive --tty janis yarn add <package name>
 
 ---
 
+## SASS Regime Guidelines (WIP)
+
+We're using a modified form of the 7-1 pattern, and BEM for CSS naming and organization
+- our css file structure and js file structure should be congruent.
+  EX. You can find css for js modules in the css/modules folder, you can find css for js page_sections in the css/page_sections folder and so on.
+- js/modules our are smallest unit and should not have any additional custom react component imports
```

Comment:
I'm not opposed to this strict definition of components in js/modules as the smallest atomic unit with no child components. But where do we put components that have children but aren't page sections.

Reply:
you know, I haven't thought this through that far, but definitely open to adding another folder called components when we need it?

Reply:
cool, i think its coming in my next PR and I'll take that suggestion as your blessing. If we wanted to align with Reactisms, we might consider the delineation not only if the component renders child components but also if the component has state or not. `js/modules` would be functional stateless components and `js/components` would be stateful components or components that render other components.

https://medium.com/@juanguardado/stateful-vs-stateless-components-444b5aa21865